vars/mavenCanaryRelease.groovy

```diff
@@ -66,3 +70,10 @@ def call(body) {
 
     contentRepository(body);
   }
+
+  def addPluginVersionTag(property, value) {
+      echo "Aadding a property <${property}> to pom.xml"
+      sh "find -type f -name 'pom.xml' | xargs sed -i -r '/<properties>/ a \\<fabric8-maven-plugin.version>${value}</fabric8-maven-plugin.version>'"
```

Comment:
I am a bit against replacing **all** pom.xml and also running find since
- it can increase the build time since it has to traverse all files.
- it can create problems later on as we are changing all pom.xml and  the source would compile for them in their local env but not in our build and this will be too hard to debug.

How about we maintain a list of files to patch? 
e.g. 

```groovy
pomXmls = [
  './pom.xml', 
  './foobar/bar/pom.xml'
]
```